src/client/components/MessageCenter/Topic/Topic.tsx

```diff
@@ -157,10 +165,12 @@ const Topic: React.FC<TopicProps> = (props) => {
           (topic.status === MessageTopicStatus.resolved &&
             (Users.isAdministrator(user) || Users.isReviewer(user, countryIso, cycle)))) && (
           <div className="topic-form">
-            <textarea
+            <TextArea
+              style={{ maxHeight: commentTextAreaMaxHeight }}
               value={message}
               placeholder={i18n.t('review.writeComment')}
               onChange={(e) => setMessage(e.target.value)}
+              rows={2}
             />
```

Comment:
Would it make sense to instead use Resizable for `<div className="topic-form">`?
And only allow resizing from top

Example:

```suggestion
<Resizable
enable={{top: true}}
className="topic-form">
            <TextArea
              style={{ maxHeight: commentTextAreaMaxHeight }}
              value={message}
              placeholder={i18n.t('review.writeComment')}
              onChange={(e) => setMessage(e.target.value)}
              rows={2}
            />
```

I like you are using the new TextArea component though

Reply:
Do we really need resizable component here ? Can we avoid it ?

Reply:
It would have conflicts with the logic that is in the TextArea, as it grows when you are typing. We could scenarios where, for example, the user lowers the size and expects it to be set, but as soon as they type it would grow again.